helm/oauth2-proxy/ci/redis-sentinel-comma-values.yaml

```diff
@@ -0,0 +1,22 @@
+sessionStorage:
+  type: redis
+  redis:
+    clientType: sentinel
+    sentinel:
+      password: "foo"
+      masterName: "mymaster"
+      connectionUrls: "redis://oauth2-proxy-redis:26379,redis://oauth2-proxy-redis:26379"
```

Comment:
The previous example was ```["redis://127.0.0.1:8000", "redis://127.0.0.1:8000"]``` so it was not a string.

Reply:
Yes I know but that is exactly the point of this PR that was never possible:

Following test are done on the current main branch:

![image](https://github.com/user-attachments/assets/aed358cb-c202-4a9c-a77d-587e7a0d92a7)

The example mentioned the following syntax:

```yaml
(e.g. `["redis://127.0.0.1:8000", "redis://127.0.0.1:8000"]`)
```

So like this:

![image](https://github.com/user-attachments/assets/467b06cb-6080-4437-8bca-aa47eac8c63f)

And even in single quotes doesn't work:
![image](https://github.com/user-attachments/assets/079a4bf9-34db-4eba-80bb-136c295c7fed)